Tests/OpenCombineTests/PublisherTests/MapTests.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  MapTests.swift
+//
+//
+//  Created by Anton Nazarov on 25/06/2019.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class MapTests: XCTestCase {
+
+    static let allTests = [
+        ("testJust", testJust),
+        ("testEmpty", testEmpty),
+        ("testError", testError),
+        ("testRange", testRange)
+    ]
+
+    func testJust() {
```

Comment:
I suggest that we don't use `Just`, `Empty` and `Fail` publishers here, and use either `PassthroughSubject` or `CustomPublisher` type instead.

The reason is that `Just`, `Empty` and `Fail` have their own specializations for the `map` operator that have nothing to do with the `Map` struct, for instance:
https://github.com/broadwaylamb/OpenCombine/blob/17f9b0a29975106d2f51d5d9b7858c5b1e612019/Sources/OpenCombine/Publishers/Publishers.Just.swift#L203-L205

This means that these tests do not actually test the `Map` type.

Reply:
Sure, thanks, didn't noticed it